test/Shared.ps1

```diff
@@ -29,7 +27,7 @@ function MakeNativePath([string]$Path) {
   $Path -replace '\\|/', [System.IO.Path]::DirectorySeparatorChar
 }
 
-[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssigments', '')]
-$module = Import-Module $PSScriptRoot\..\posh-sshell.psd1
+# [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssigments', '')]
+# $module = Import-Module $PSScriptRoot\..\posh-sshell.psd1
```

Comment:
Go ahead and delete this. Shouldn't be necessary since this module doesn't need posh-git's `prompt` stuff.